conf/bafu.conf.part

```diff
@@ -55,6 +55,24 @@ source src_ch_bafu_fauna_wildtierkorridor_national : def_searchable_features
         from fauna.wildtierkorridore_national
 }
 
+source src_ch_bafu_unesco_weltnaturerbe : def_searchable_features
+{
+    sql_db = bafu_${DBSTAGING}
+    sql_query = \
+        SELECT bgdi_id as id \
+            , coalesce(name,'') as label \
```

Comment:
for single attributes in the label the coalesce is not necessary, i will update the wiki.

the coalesce was originally introduced to prevent :

```sql
attribute1 || attribute2 as test -- will be NULL if one of the two attributes is NULL
coalesce(attribute1,'') || coalesce(attribute2,'') as test -- wont be NULL if one of the attributes is NULL

-- the same can be done with:

concat(attribute1, ' ', attribute2)
-- Unlike the concatenation operator ||, the CONCAT function ignores NULL arguments.
```